test/SingleFileTestApp/SingleFileTestApp.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <IsTestProject>false</IsTestProject>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <LangVersion>11</LangVersion>
+    <ImplicitUsings>true</ImplicitUsings>
```

Comment:
We do... for this particular project (which isn't really a "Test" project per say) it has it's own `Directory.Build.props` file that intentionally does not import the parent.

```
<Project DefaultTargets="Build;Publish">
  <!-- Ignore Directory.Build.props from parent -->
</Project>
```

The reason being that this isn't really a Test project - it simply creates an artefact that's used by some of the tests. If it inherited all the same Build props as the other test projects, it would need an unnecessary reference to the Sentry project... which would simply slow down building/publishing this single file app. 

Ultimately it's a Hello World app, so I figured if it sits forever on LanVersion 11, it's no biggy.